### PR TITLE
Log distribution of containers in density test

### PR DESCRIPTION
**Pull request by gmarek**
Ref. #22072

cc @kubernetes/sig-scalability

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -314,8 +314,30 @@ var _ = Describe("Density", func() {
...
+				nodeNames = append(nodeNames, k)
+			}
+			sort.Strings(nodeNames)
+			for _, node := range nodeNames {
+				Logf("%v: %v pause pods, system pods: %v", node, pausePodAllocation[node], systemPodAllocation[node])
+			}
+
 			if itArg.runLatencyTest {
-				Logf("Schedling additional Pods to measure startup latencies")
+				By("Schedling additional Pods to measure startup latencies")
```
Comment (wojtek-t) [inline test/e2e/density.go]: nit/Schedling/Scheduling/

**Comment by wojtek-t** [comment]
Just one nit

**Comment by gmarek** [comment]
Done. PTAL

**Comment by wojtek-t** [comment]
LGTM - thanks

**Pull request merged**